src/main/java/com/liferay/ide/idea/ui/modules/springmvcportlet/LiferaySpringMvcPortletBuilder.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.ide.idea.ui.modules.springmvcportlet;
+
+import com.intellij.ide.util.projectWizard.ModuleBuilder;
+import com.intellij.ide.util.projectWizard.ModuleWizardStep;
+import com.intellij.ide.util.projectWizard.WizardContext;
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.externalSystem.service.execution.ProgressExecutionMode;
+import com.intellij.openapi.externalSystem.util.ExternalSystemUtil;
+import com.intellij.openapi.module.ModuleType;
+import com.intellij.openapi.module.StdModuleTypes;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.roots.ModifiableRootModel;
+import com.intellij.openapi.util.io.FileUtilRt;
+import com.intellij.openapi.vfs.LocalFileSystem;
+import com.intellij.openapi.vfs.VirtualFile;
+
+import com.liferay.ide.idea.util.BladeCLI;
+import com.liferay.ide.idea.util.LiferayWorkspaceSupport;
+
+import icons.LiferayIcons;
+
+import java.io.File;
+
+import javax.swing.Icon;
+
+import org.jetbrains.plugins.gradle.util.GradleConstants;
+
+/**
+ * @author Terry Jia
+ */
+public class LiferaySpringMvcPortletBuilder extends ModuleBuilder implements LiferayWorkspaceSupport {
```

Comment:
Lets rename this `SpringMVCPortletModuleBuilder` so that we know we are building a `module` and not `project` since workspaces arre projects and projects are modules in intellij.